.github/workflows/ci.yml

```diff
@@ -334,3 +332,20 @@ jobs:
       - name: Push load test result
         if:  ${{ success() && github.ref_name == 'v3'}}
         run: git push 'https://github-actions:${{ secrets.GITHUB_TOKEN }}@github.com/nginx/agent.git' benchmark-results:benchmark-results
+  publish-packages:
+    name: Publish NGINX Agent v3 packages
+    if: ${{ github.event_name =='pull_request' &&
+            github.base_ref == 'v3' &&
+            github.event.pull_request.merged == true &&
+            !github.event.pull_request.head.repo.fork }}
+    needs: [ lint, unit-test, component-test, performance-test,
```

Comment:
Think you need to remove `component-test`
The pipeline is failing for this:
```
The workflow is not valid. .github/workflows/ci.yml (Line: 341, Col: 31): Job 'publish-packages' depends on unknown job 'component-test'.
```

Reply:
Good catch, did you run the pipeline manually to view this error?

Reply:
well, I see that now in the annotations actions